linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/TenantConfigrationRestfulApi.java

```diff
@@ -240,11 +239,7 @@ public Message checkUserCreator(
       if (!Configuration.isAdmin(userName)) {
         return Message.error("Failed to check-user-creator,msg: only administrators can configure");
       }
-      checkResult = tenantConfigService.isExist(user, creator);
-      // The incoming id represents the update. The update allows user-create exist
-      if (StringUtils.isNotBlank(id)) {
-        checkResult = !checkResult;
-      }
+      checkResult = !tenantConfigService.isExist(user, creator);
```

Comment:
with`!`?

Reply:
yes，When creating tenant query whether the label exists. The existence check result should be false